index.bs

```diff
@@ -5222,13 +5210,14 @@ Member Submissions</h2>
 				<li>
 					If <dfn>acknowledged</dfn>,
 					the Team <em class="rfc2119">must</em> make the [=Member Submission=] available
-					at the public W3C website.
-					The list of <a href="https://www.w3.org/submissions/">acknowledged Member Submissions</a> [[SUBMISSION-LIST]]
-					is available at the W3C website.
+					at the public W3C website
+					alongside other <a href="https://www.w3.org/submissions/">acknowledged Member Submissions</a>.
+					[[SUBMISSION-LIST]]
 
-				<li>If <dfn>rejected</dfn>,
+				<li>
+					If <dfn>rejected</dfn>,
 					the [=Team=] <em class=rfc2119>must</em> inform the [=Advisory Committee representative=](s) of the [=Submitter=](s),
```

Comment:
`[=Advisory Committee representative(s)=]` synonym also needed.
```suggestion
					the [=Team=] <em class=rfc2119>must</em> inform the [=Advisory Committee representative(s)=] of the [=Submitter(s)=],
```

Reply:
Filed https://github.com/speced/bikeshed/issues/2948. If/when bikeshed addresses that, I'm happy to go along with your suggestion, but until then, that seems like too much work for not enough benefit.